READMEs/developers.md

```diff
@@ -1,14 +1,16 @@
 # Developing ocean.py
 
 This README is how to further *develop* ocean.py. (Compare to the quickstarts which show how to *use* it.)
-
 Steps:
 1. **Install dependencies**
-1. **Start blockchain service** (only needed for ganache)
-1. **Deploy** the contracts to {local, rinkeby, mainnet}
-1. **Test**
+2. **Configure the services**
```

Comment:
Note that there's a recent change to this file, as of a couple hours ago, now in `master`. You'll need to merge that into this branch before merging this branch into `master`. It will add an additional step in this list, and below. Make sure that all the numbers line up in the summary list, and below.